.nix/buffrs.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs, crane, rustToolchain, system, advisory-db, buildInputs
```

Comment:
please can we not put nix stuff in a hidden folder?

Reply:
moved to a normal `nix` folder